README.adoc

```diff
@@ -68,45 +70,46 @@ $ cd my_openshift/fuse-springboot-circuit-breaker-booster
 $ mvn clean -DskipTests fabric8:deploy -Popenshift
 ----
 
-. If your environment dose not support automatic sidecar injection, you need to manually inject istio sidecar envoy proxy:
+. If your environment does not support automatic sidecar injection, you must manually inject the Istio sidecar envoy proxy:
 +
 [source,bash,options="nowrap",subs="attributes+"]
 ----
-$ oc get dc name-service -o yaml | /home/valdar/Downloads/istio-1.0.2/bin/istioctl kube-inject -f - | oc apply -f -
+$ oc get dc name-service -o yaml | /home/username/Downloads/istio-1.0.2/bin/istioctl kube-inject -f - | oc apply -f -
```

Comment:
probably here is better to just use `$ oc get dc name-service -o yaml | <istio_install_directory>/bin/istioctl kube-inject -f - | oc apply -f -` ?